addons/account/models/account_move.py

```diff
@@ -2042,6 +2045,13 @@ def _compute_next_payment_date(self):
         for move in self:
             move.next_payment_date = min([line.payment_date for line in move.line_ids.filtered(lambda l: l.payment_date and not l.reconciled)], default=False)
 
+    @api.depends('line_ids.tax_ids', 'fiscal_position_id')
+    def _compute_incorrect_taxes_warning(self):
+        for move in self:
+            move.display_incorrect_taxes_warning = (
+                move.fiscal_position_id and not move._validate_tax_alignment_with_fpos()
```

Comment:
```suggestion
                move.is_invoice() and move.fiscal_position_id and not move._validate_tax_alignment_with_fpos()
```
you forgot to filter out the right move

Reply:
Done :+1: